### ISSUE TITLE
User login using e-mail alias

### ISSUE THREAD

**Issue by ilmartyrk** [citizenos/citizenos-api#167]
**What is the problem?**
When user is added to a topic using an email alias, he/she could possibly not access the topic as google returns the original e-mail

**Why is this a problem?**
When someone is trying to invite somebody to a topic and doesn't know the correct e-mail (User publicly uses alias for e-mail), the user could possibly not access topic as login returns original e-mail.

**Comment by loorm** [comment]
Triage 21. Need to clarify the use case in order to reproduce problem. @ilmartyrk - what do you mean by "alias"? Please tag me in reply. 

Moved to "in prep".

**Comment by ilmartyrk** [comment]
@loorm for example if I have google e-mail ilmar.tyrk@gmail.com, google also has aliases for this e-mail eg. if you send mail to ilmartyrk@gmail.com or i.lmartyrk@gmail.com it will still be delivered to my mailbox. If user is added to a topic using by alias, for my example ilmartyrk@gmail.com instead of ilmar.tyrk@gmail.com I will still get the invite to my ilmar.tyrk e-mail, but I won't be able to access the topic as my user e-mail in our system isn't matching the alias.

**Comment by loorm** [comment]
@ilmartyrk Got it, thanks!

Reassigning for Triage.

**Comment by KatiVellak** [comment]
@ilmartyrk How can we be sure that it was correct "alias"? Maybe there is a katrin.vellak and kati.vellak and somebody misspells and writes katri.vellak and wrong person receives the invite.

**Comment by loorm** [comment]
Triage 22. This needs further thinking about a solution. Related to invites and join links. Back to in preparation.

**Comment by loorm** [comment]
I've thought about it. Assigning this for Triage. Don't see a reason to keep this as blocked. Two possible decisions, but both lead to closing this issue:

1. We don't do anything. Grown-up people are responsible for inputting correct e-mail addresses. I checked - when I receive the invite to an alias and then try to follow the link in the e-mail (while being logged in with my correct e-mail), the dialog window comes up and says "Log in with (alias e-mail) to accept invitation. So user can easily see, that the invitation has been sent to the wrong e-mail address and can ask whoever sent it to send another invitation to the correct e-mail address. + the problems that Kati mentioned above. If we decided we won't do anything, let's unblock and close this issue.

2. We add a feature, where user can "request access to a topic". This is already a separate, open issue https://github.com/citizenos/citizenos-fe/issues/289. If we decide to add the feature, we still need to unblock and close this issue, because the other issue already exists.
